src/views/Home.tsx

```diff
@@ -10,26 +11,28 @@ interface Props {
 }
 
 export function Home({ data, setListPath, user }: Props) {
+	if (!user) {
```

Comment:
I wonder if we could share a similar pattern that the nav bar components used? I like the ternary that returns one or the other. You could have something like this

`user ? <AuthenticatedHome/> : <UnauthenticatedHome/>`

Reply:
+1 !